### PR TITLE
Restructure the project directory structure and build procedure

### PR DESCRIPTION
**Pull request by bhufmann**
- Move the vscode extension into directory vscode-trace-extension
- Move react app running in webview into directory vscode-trace-webviews
- Use lerna to build
- Build the extension with command "yarn"
- README updates
- Update Copyright information
- Some minor clean-ups

Signed-off-by: Bernd Hufmann <Bernd.Hufmann@ericsson.com>

### CODE REVIEW

**Review by tahini — CHANGES_REQUESTED** [review]
This is awesome!! No more symlinking scripts! Thanks for finding this.

The 2 workspaces work great too!

Just a small comment about the yarn link commands which do not work exactly as described in the README here

File: README.md [hunk 1]
```diff
@@ -1,37 +1,43 @@
...
 The code was migrated from the [PR in theia-trace-extension](https://github.com/theia-ide/theia-trace-extension/pull/124)
 
-It depends on the trace viewer plugins from the [theia trace extension package](https://github.com/theia-ide/theia-trace-extension/). So it needs to be cloned and built before building the vscode extension.
+It depends on the trace viewer plugins from the [theia trace extension package](https://github.com/theia-ide/theia-trace-extension/) and the [tsp typescript client](https://github.com/theia-ide/tsp-typescript-client/). So it needs to be cloned, built and linked to the `vscode-trace-extension` before building the vscode extension
+
+```
+git clone git@github.com:theia-ide/tsp-typescript-client.git
+cd tsp-typescript-client
+yarn
+yarn link tsp-typescript-client
```
[the remark this answers is not in the document]
Comment (bhufmann) [reply README.md:15]: Yes, true. I'll update it.

File: README.md [hunk 2]
```diff
@@ -1,37 +1,43 @@
...
+```
 
 ```
 git clone git@github.com:theia-ide/theia-trace-extension.git
 cd theia-trace-extension
+yarn link "tsp-typescript-client"
 yarn
+yarn link "@trace-viewer/base"
```
[the remark this answers is not in the document]
Comment (bhufmann) [reply README.md:23]: I'll update

**Comment by bhufmann** [comment]
> This is awesome!! No more symlinking scripts! Thanks for finding this.
> 
Behind the scenes it actually does symlinks for you. BTW, you can use it to run libraries from source code (e.g. use local timeline-chart in theia-trace-extension). Don't forget to yarn unlink <package> when not needed anymore.

**Review by tahini — APPROVED** [review]
Great addition, thank you!

**Review by MatthewKhouzam — APPROVED** [review]
This looks good to me.

**Pull request merged**